app/views/editions/_actions.html.erb

```diff
@@ -4,10 +4,14 @@
 
       <% edition_policy = EditionPolicy.new(current_user, edition) %>
 
-      <% if edition_policy.can_preview? %>
+      <% if edition_policy.can_preview? && edition_policy.can_discard_draft? %>
```

Comment:
I'm not sure about changing this as it breaks the method naming convention from all the others (they all start with 'can')...what do you think @thehenster ?